Additional information for SGB 4712

This directory contains: 
1. the genome fasta file
2. checkM genome quality assessment (98.73% completeness, 0% contamination)
3. prokka protein annotation (.faa protein fasta file and .gff protein annotation file)
4. eggnog functional annotation (.tsv)

The closest reference is GCF_003478505.1 with 98.83% ANI (average nucleotide identity) and 89.93% AF (alignment fraction)
https://gtdb.ecogenomic.org/genomes?gid=GCF_003478505.1
(GTDB-tk - Genome Taxonomy Database)

GTDB Taxonomy: d__Bacteria; p__Firmicutes_A; c__Clostridia; o__Lachnospirales; f__Lachnospiraceae; g__UBA9502; s__UBA9502 sp003478505
NCBI Taxonony: d__Bacteria; x__Terrabacteria group; p__Firmicutes; c__Clostridia; o__Clostridiales; f__Clostridiaceae; g__Clostridium; x__unclassified Clostridium; s__Clostridium sp. AF27-2AA

Methods
Genome assembly quality was assessed with CheckM v1.0.18 [1]. 
Taxonomic classification was assigned based on the Genome Taxonomy Database (GTDB release 05-RS95) [2] using the toolkit GTDB-tk v1.3.0 [3]. 
The average nucleotide identity (ANI) and alignment fraction between the genome assembly and the identified reference genome were computed with fastANI v1.31 [4]. 
Protein coding sequences were annotated with Prokka v1.14 [5] and functional annotation of the protein coding sequences was performed with eggNOG-mapper v2.0 [6]. 
 
1. Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P., & Tyson, G. W. (2015). CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. Genome research, 25(7), 1043-1055. 
 
2. Parks, D. H., Chuvochina, M., Chaumeil, P. A., Rinke, C., Mussig, A. J., & Hugenholtz, P. (2020). A complete domain-to-species taxonomy for Bacteria and Archaea. Nature biotechnology, 38(9), 1079-1086. 
 
3. Chaumeil, P. A., Mussig, A. J., Hugenholtz, P., & Parks, D. H. (2020). GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database. Bioinformatics, 36, 1925–1927 (2020). 
 
4. Jain, C., Rodriguez-R, L. M., Phillippy, A. M., Konstantinidis, K. T., & Aluru, S. (2018). High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries. Nature communications, 9(1), 1-8. 
 
5. Seemann, T. (2014). Prokka: rapid prokaryotic genome annotation. Bioinformatics 30, 2068–2069  
 
6. Huerta-Cepas, J., Forslund, K., Coelho, L. P., Szklarczyk, D., Jensen, L. J., Von Mering, C., & Bork, P. (2017). Fast genome-wide functional annotation through orthology assignment by eggNOG-mapper. Molecular biology and evolution, 34(8), 2115-2122. 

